### PR TITLE
Bugfix/show created tweet

### PR DESCRIPTION
**Pull request by Barylo**
added logic to post a tweet after creation, fixed some bugs

### CODE REVIEW

File: src/components/TweetCreationForm.tsx [hunk 1]
```diff
@@ -44,6 +44,7 @@ const StyledPopover = styled(Popover)`
 
 const TweetCreationForm: React.FC = () => {
   const [content, setContent] = useState("");
+  const [show, setShow] = useState(true);
```
Comment (VitaliiDenysiuk) [inline src/components/TweetCreationForm.tsx]: why did you add "show" if you never use it for visualization?
I did not find any place where you use it at UI

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]
I did not understand what you try to do.
Your task:
1) currently, we're sending request to backend, to create tweet, but nothing happen after that. Expected behavior is to show created tweet after receiving response from backend
2) close tweet creation modal after successful creating tweet

**I checked your branch: no task worked at all**
maybe you forgot push something

File: src/components/TweetCreationForm.tsx (hunk 1 above)
Comment (Barylo) [reply src/components/TweetCreationForm.tsx]: Yup it is unused code - it was a try, but with no success. deleted. Alex said that for now it is impossible to get done the second part of the task - to let the modal close after click, so he let it be unsolved till some future changes in architecture of the app.

**Review by alexandrtovmach — APPROVED** [review]
